### PR TITLE
Feature/grails5

### PR DESCRIPTION
**Pull request by sughics**
PR for https://github.com/AtlasOfLivingAustralia/dashboard/issues/119

### CODE REVIEW

File: src/main/groovy/au/org/ala/dashboard/Constants.groovy [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+import grails.core.GrailsApplication
+
 /**
  * Created by rui008 on 26/11/2014.
  */
 class Constants {
 
     class WebServices {
 
+        def grailsApplication
```
Comment (sbearcsiro) [inline src/main/groovy/au/org/ala/dashboard/Constants.groovy:12]: Is this used?

File: build.gradle [hunk 2]
```diff
@@ -6,19 +6,18 @@ buildscript {
...
 plugins {
- //   id 'org.springframework.boot' version '1.4.2.RELEASE'
-    // Add Git properties plugin.
-    id 'com.gorylenko.gradle-git-properties' version '1.4.17'
+    id "com.gorylenko.gradle-git-properties" version "2.4.1"
 }
 
 
-version "2.4"
+version "2.4.1-SNAPSHOT"
```
Comment (sbearcsiro) [inline build.gradle:20]: We should probably bump the minor or major version.

**Review by sbearcsiro — COMMENTED** [review]
Looks good - just looks like Constants.WebServices had `def grailsApplication` accidentally added.

File: build.gradle (hunk 2 above)
Comment (sughics) [reply build.gradle:20]: Agreed. Will bump up the minor version `2.5.0` as I've done with some other apps

File: src/main/groovy/au/org/ala/dashboard/Constants.groovy (hunk 1 above)
Comment (sughics) [reply src/main/groovy/au/org/ala/dashboard/Constants.groovy]: Updated

**Review by sbearcsiro — APPROVED** [review]
LGTM

**Pull request merged**